code/modules/economy/economy_misc_vr.dm

```diff
@@ -0,0 +1,13 @@
+/hook/startup/proc/pay_raise()
+	economic_species_modifier = list(
```

Comment:
If you're making them all equal just remove it entirely or make it a not list and change the line in job.dm to not use the fancy species stuff.